app/controllers/users_controller.rb

```diff
@@ -26,7 +26,7 @@ def feed
     
     feed_likes = Like.all.where(fan_id: leader_ids).order(created_at: "asc")
     
-    if feed_likes.nil?
+    if feed_likes.count == 0
```

Comment:
You could use `.empty?` or `.zero?` here as well.

```suggestion
    if feed_likes.empty?
```